packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -11,7 +11,7 @@ $detailsList-item-selected-hover-color: $listItemSelectedHoverColor;
 .root {
   @include focus-border();
 
-  display: inline-block;
+  display: flex;
```

Comment:
is this flex required? it seems that this should work with just an update to the fields.